simplq/src/styles/homePage.module.scss

```diff
@@ -50,4 +50,11 @@ div {
   @include text-color();
   @include center-vertically();
   @include center-horizontally();
+  div {
```

Comment:
These styles will apply to divs globally. Just mentioning. 

Also, I didn’t understand the min width part 🤔

Reply:
Without the min-width, when I centre it, it was coming like this, as the length of the text varies: 

![image](https://user-images.githubusercontent.com/11256376/89076784-91a49f80-d39e-11ea-98e7-c8301287d584.png)